### PR TITLE
Discovered Tests from Wildfly

### PR DESCRIPTION
**Pull request by average-avancha**
Adding list of discovered tests from Wildfly. 

Logs can be found on /home/rra2/new-tests/wildfly-galleon-pack-layer-tests.nondex_log on my VM.

### CODE REVIEW

**Comment by average-avancha** [comment]
I ran each test individually locally on my machine, and each test fails for some NonDex seeds but not all. I can replicate the testing logs on my VM.

**Comment by darko-marinov** [comment]
Please run on VM. We can't see logs on your machine. You likely want to use a script for this or will be very slow. You can look at scripts already in IDoFT or pending PRs, or you can ask on Campuswire.

**Comment by average-avancha** [comment]
I've managed to replicate the work on the VM.

Path to individual logs for each of the test runs with NonDex on the VM:
/home/rra2/flaky_testing/wildfly_feature_pack_tests_fix/galleon-pack

The tests with higher batch sizes on the VM are wrapping up to provide more support that the tests are flaky depending on the seed.

**Comment by darko-marinov** [comment]
Please squash.

**Comment by average-avancha** [comment]
> Please squash.

Done.

**Comment by darko-marinov** [comment]
@zzjas please check logs.

File: pr-data.csv [hunk 1]
```diff
@@ -5911,6 +5911,29 @@ https://github.com/wildfly/wildfly,cb7af61bfbb5615b522cabf8ded9a135178089b7,unde
...
 https://github.com/wildfly/wildfly,b19048b72669fc0e96665b1b125dc1fda21f5993,undertow,org.wildfly.extension.undertow.UndertowSubsystemTestCase.testRuntime,ID,,,
+https://github.com/wildfly/wildfly,f9b089fcdfb089eabcf68117e6141af3be826d21,galleon-pack/layer-metadata-tests,org.wildfly.feature.pack.layer.tests.microprofile.telemetry.MicroProfileTelemetryLayerMetaDataTestCase.org.wildfly.feature.pack.layer.tests.microprofile.telemetry.MicroProfileTelemetryLayerMetaDataTestCase.org.wildfly.feature.pack.layer.tests.microprofile.telemetry.MicroProfileTelemetryLayerMetaDataTestCase.testClassInSdkTraceSamplersPackage,ID,,,
+https://github.com/wildfly/wildfly,f9b089fcdfb089eabcf68117e6141af3be826d21,galleon-pack/layer-metadata-tests,org.wildfly.feature.pack.layer.tests.microprofile.telemetry.MicroProfileTelemetryLayerMetaDataTestCase.org.wildfly.feature.pack.layer.tests.microprofile.telemetry.MicroProfileTelemetryLayerMetaDataTestCase.testClassInSdkAutoConfigureSpiLogsPackage,ID,,,
```
Comment (zzjas) [inline pr-data.csv:5915]: These two test names include repetitive paths.

**Review by zzjas — COMMENTED** [review]
Logs are ok for all tests, but two test names are wrongly written.

**Comment by zzjas** [comment]
Ok to merge @darko-marinov

**Comment by darko-marinov** [comment]
Checker is failing.

Seemingly "repetitive" tests may be how JUnit reports failures in test class constructor. Would need to inspect more.

**Comment by average-avancha** [comment]
Fixed format checker ordering issue caused from fixing tests with repetitive paths.

**Comment by darko-marinov** [comment]
Let's accept this for now and check the details when more fixing is done.

**Pull request merged**